simexpal/base.py

```diff
@@ -1251,7 +1251,7 @@ def _update_status_cache_dict(self):
 				status = Status.TIMEOUT
 			elif status_dict['signal']:
 				status = Status.KILLED
-			elif status_dict['status'] > 0:
+			elif status_dict['status'] > 0 or status_dict['error']:
```

Comment:
What if `error` does not exist (e.g., because we're running on a .status file that was generated by an older version)?

Reply:
I changed it to ``status_dict.get('error', None)`` to handle this case.